src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,92 @@
-export const Pagination = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: FC<Props> = ({
+  total,
+  perPage,
+  onPageChange,
+  currentPage = 1,
+}) => {
+  const paginationLength = Math.ceil(total / perPage);
+  const paginationItems = Array(paginationLength).fill(0).map((_, i) => i + 1);
+  const prevIsDisabled = currentPage === 1;
+  const nextIsDisabled = currentPage === paginationLength;
+
+  const changePageToPrev = () => {
+    if (!prevIsDisabled) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const changePageToNext = () => {
+    if (!nextIsDisabled) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: prevIsDisabled,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={prevIsDisabled}
+          onClick={changePageToPrev}
+        >
+          «
+        </a>
+      </li>
+
+      {paginationItems.map(item => (
+        <li
+          className={cn('page-item', {
+            active: item === currentPage,
+          })}
+          key={uuidv4()}
```

Comment:
I don't know why u use here uuidv4, I think page number is always unique key name

Reply:
Probably yes, also was thinking about that, but just for more "safety" of not getting reject I'm added uuid :D